docs/section21/Rule21-5.md

```diff
@@ -0,0 +1,65 @@
+
+# Boiler - Rule 21-5  
+
+**Rule ID:** 21-5  
+**Rule Description:** The baseline building design boiler plant shall be modeled as having a single boiler if the baseline building design plant serves a conditioned floor area of 15,000sq.ft. or less, and as having two equally sized boilers for plants serving more than 15,000sq.ft.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. Baseline number of HW Loops > 0 and number of boilers in HW loop > 0.  
+
```

Comment:
Few thoughts:
1. The rule should have rule dependency that baseline HVAC system type was established correctly (i.e., there should indeed be a boiler in the baseline). 
2. I think having more than one hot water loop in the baseline should be flagged as an error.

Reply:
as discussed on the RDS call - need to have a separate RDS to verify that the baseline HVAC is established correctly. SG will upload the TCDs for baseline HVAC checks and Xing will update the applicability check to verify the baseline system typee.

Reply:
same question as 21-3, 'B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12.' is a little confusing. I assume Type-1, 5, 7, 11, 12 are the typical baseline systems and 'Type-1a, 7, 11, 12' are the modified baseline systems which have PHW or PCHW in proposed? If yes, we should instead use terminology 'Baseline System Type based on G3.1.1.3.3.' @mkarpman to confirm.

Reply:
Yes, I agree